config.example/group_vars/slurm-cluster.yml

```diff
@@ -187,3 +187,12 @@ slurm_enable_monitoring: true
 # Inventory host groups where cluster monitoring services will be installed
 # (Prometheus, Grafana, etc)
 slurm_monitoring_group: "slurm-master"
+
+################################################################################
+# Logging with rsyslog                                                         #
+################################################################################
+slurm_enable_rsyslog_server: true
+slurm_enable_rsyslog_client: true
+rsyslog_server_hostname: "{{ groups['slurm-master'][0] }}"
+rsyslog_client_tcp_host: "{{ rsyslog_server_hostname }}"
+rsyslog_client_group: "slurm-node"
```

Comment:
Any reason this isn't slurm-cluster

Reply:
Only to avoid redundant logging by having the head node forward to itself! But shouldn't be any issues if we want to enable that, just a little extra disk usage.

Reply:
I only ask because in the above you used k8s-cluster instead of kube-node

Reply:
That's a good point 😄 edited this to align with the k8s config